### ISSUE TITLE
Название площадки

### ISSUE THREAD

**Issue by a-matyukh** [easably/games#26]
*"Tutor" часто требует осмысления и траты времени на понимание сути продукта*

*самое безопасное название*
Games
*ближе всего к сути площадки*
League
Yard
Gym
Playground

- [ ] Определиться с название
- [ ] Имплементить
      - [ ] Переименовать title страницы

**Comment by a-matyukh** [comment]
*самое безопасное название*
Games

*ближе всего к сути площадки*
League
Yard
Gym
Playground

**Comment by sergeifedorenko** [comment]
@a-matyukh @EugeneShastak 
Будем пробоваать решать проблему через грамматические уроки 

@alexnpol FYI

**Comment by EugeneShastak** [comment]
@sergeifedorenko, согласен, если хорошо объяснить что это- тогда все поймут и привыкнут
